tests/snmp/test_snmp_phy_entity.py

```diff
@@ -428,7 +441,7 @@ def _get_transceiver_sensor_data(duthost, name):
 
 
 @pytest.mark.disable_loganalyzer
-def test_turn_off_pdu_and_check_psu_info(duthost, localhost, creds, pdu_controller):
+def test_turn_off_psu_and_check_psu_info(duthosts, enum_rand_one_per_hwsku_hostname, snmp_physical_entity_info, localhost, creds, pdu_controller):
```

Comment:
Currently the `pdu_controller` definition is like below:
```
@pytest.fixture(scope="module")
def pdu_controller(duthosts, rand_one_dut_hostname, conn_graph_facts, pdu):
```
So, the `pdu_controller` object used by this test case could be a PDU controller for other node in current test bed. How to address this issue?